sdk/cognitivelanguage/ai-language-text/test/public/expectations.ts

```diff
@@ -8696,7 +8696,7 @@ export const expectation71: any = [
         entityRelations: [],
         id: "0",
         warnings: [],
-        detectedLanguage: { "0": "e", "1": "n" },
+        detectedLanguage: { iso6391Name: "en" },
```

Comment:
I'm thinking it might be good to mock a test for when the service eventually starts returning the object for detectedLanguage

Reply:
I don't have infrastructure to mock the service at the moment, but I added unit tests for the mitigation utility function in https://github.com/Azure/azure-sdk-for-js/pull/23720/commits/c5fc72b0978987abcf2caef48aad457e45a089cd